internal/cobraprompt/prompt.go

```diff
@@ -146,7 +164,11 @@ func (co CobraPrompt) Run(ctx context.Context, root *cobra.Command, cnfg *hazelc
 			root, _ = rootcmd.New(cnfg)
 			prepareRootCmdForPrompt(co, root)
 			root.SetArgs(promptArgs)
-			if err := root.ExecuteContext(ctx); err != nil {
+			err = root.ExecuteContext(ctx)
+			if _, writeErr := f.WriteString(fmt.Sprintln(in)); writeErr != nil {
+				root.Printf("Could not persist command to history:%s\nErr:%s\n", in, writeErr)
```

Comment:
Similarly, IMO no need to show a message if we can't write the history.

Reply:
Replaced it with a todo https://github.com/hazelcast/hazelcast-commandline-client/pull/66/commits/fe6bf1e1b2ca7dd456aff72357e77644aca95af7